src/charm.py

```diff
@@ -78,96 +53,53 @@ def __init__(self, *args):
 
         self.dashboard_provider = GrafanaDashboardProvider(self)
 
+        self._container_name = "dex"
```

Comment:
Would `dex-auth` be more appropriate?

Reply:
I wondered that at first, but looking at their website etc I don't see them using "auth".  Did we make that up?

Reply:
We probably made that up, lets stick with dex.